### PR TITLE
adds unittest for get_input

### PR DESCRIPTION
**Pull request by nitish-tripathi**
Adds some more unittests for get_input method, Yipee!

### CODE REVIEW

File: tests/tests_core.py [hunk 1]
```diff
@@ -7,14 +7,63 @@
...
+    def test_get_inputs_givenInputJson_paramFileIsNotPartOfInputs(self):
+        os.environ["WFE_INPUT_JSON"] = '{"WFE_output_params_file":"param.json","red":0.2}'
+        core = _core._core()
+
+        inputs = core._get_inputs()
+        
+        self.assertFalse(hasattr(inputs, 'WFE_output_params_file'))
+        self.assertEqual(len(vars(inputs)), 1)
+
+    def test_get_inputs_givenDecimel_isDeserialized(self):
```
Comment (thomasirmer) [inline tests/tests_core.py]:
```suggestion
    def test_get_inputs_givenDecimal_isDeserialized(self):
```

File: apeer_dev_kit/_core.py [hunk 2]
```diff
@@ -18,8 +22,8 @@ def _get_inputs(self):
             self.inputs = Namespace(**self._input_json)
             return self.inputs
         except KeyError:
-            print(
-                "Environment variable WFE_INPUT_JSON not found. Please add WFE_INPUT_JSON as an environment variale. Example of WFE_INPUT_JSON: {\"WFE_output_params_file\":\"wfe_module_params_1_1.json\", \"red\":0, \"green\":0.5, \"blue\":0}")
+            raise KeyError(
+                'Environment variable WFE_INPUT_JSON not found. Please add WFE_INPUT_JSON as an environment variale to get inputs')
```
Comment (thomasirmer) [inline apeer_dev_kit/_core.py]: Shouldn't this error message refer to WFE_output_params_file not found? --because that what could hapepen in the try.

Comment (nitish-tripathi) [reply apeer_dev_kit/_core.py]: yups, copy paste error

File: tests/tests_core.py [hunk 3]
```diff
@@ -7,14 +7,70 @@
...
+            core._get_inputs()
+
+    def test_get_inputs_givenInputJson_paramFileIsNotPartOfInputs(self):
+        os.environ["WFE_INPUT_JSON"] = '{"WFE_output_params_file":"param.json","red":0.2}'
+        core = _core._core()
+
+        inputs = core._get_inputs()
+        
+        self.assertFalse(hasattr(inputs, 'WFE_output_params_file'))
+        self.assertEqual(len(vars(inputs)), 1)
```
Comment (thomasirmer) [inline tests/tests_core.py]: would remove that since it's not testing what the test says. the previous assert is enough

Comment (thomasirmer) [inline tests/tests_core.py]:
```suggestion
```

Comment (nitish-tripathi) [reply tests/tests_core.py]: this asserts the logic that `WFE_output_params_file` is not part of `inputs`. I would not remove it.

Comment (nitish-tripathi) [reply tests/tests_core.py]: No as I have commented below, this asserts the logic that WFE_output_params_file is not part of inputs. I would not remove it.

**Pull request merged**